### DISCUSSION TITLE
How to get the Open Sourcerer and Heart on Your Sleeve achievements?

### DISCUSSION THREAD

**Discussion by Chameleon-Coder** [Schweinepriester/github-profile-achievements#574] [Q&A]
title :tm:

**Comment by Schweinepriester** [comment]
[Currently unknown, probably not possible for now](https://github.com/Schweinepriester/github-profile-achievements/issues/478).

**Reply by Chameleon-Coder** [reply]
Thanks.

**Comment by Tynab** [comment]


**Reply by Schweinepriester** [reply]
Source? Or a link to a user who has Open Sourcerer already?

Sounds like you asked a chatbot ;)

**Reply by xamidi** [reply]
This was clearly written by AI. He spammed the same garbage in [community/discussions](https://github.com/orgs/community/discussions/44972#discussioncomment-5011640).

**Reply by castilla204** [reply]
hello my friend

**Comment by Tynab** [comment]
To earn the Heart on Your Sleeve achievement on GitHub, you need to publicly share a project on GitHub that contains a valid license and a README file. Here are the steps you can follow:
1. Sign up for a GitHub account if you haven't already.
2. Create a new project or select an existing project that you want to share on GitHub. The project should include a valid open source license and a README file that provides information about the project.
3. Create a new repository on GitHub and upload your project files.
4. Add a description and any other relevant information about your project, such as installation instructions or usage examples, to the README file.
5. Choose an appropriate open source license for your project and include a LICENSE file in the repository. GitHub provides several options for open source licenses that you can choose from.
6. Make sure that the repository is set to "public" so that others can view and access your project.
7. After you have completed these steps, you should receive the Heart on Your Sleeve badge on your GitHub profile.
Note that GitHub has specific requirements for what constitutes a valid open source license, so be sure to review the license options and choose one that fits the needs of your project. Additionally, it's important to provide clear and comprehensive information about your project in the README file to help others understand how to use and contribute to your project. Good luck!

**Reply by Schweinepriester** [reply]
@Tynab last warning before I block you. this one stays: https://github.com/Schweinepriester/github-profile-achievements/discussions/574#discussioncomment-5011674

**Comment by locthdzev** [comment]
Hey all! I’ve been digging around about these two badges, "Open Sourcerer" and "Heart on Your Sleeve", and heres what i found from poking around the community and github stuff:

Open Sourcerer
- What it is? Its about helping out open-source projects with pull requests (prs).
- How to get: 
  + Make some PRs to diffrent public repos on github—not just one place.
  + Get them merged by the maintainers. Could be fixing bugs, adding stuff, or even docs tweaks.
  + Heads up: it’s still “in testing” (as of march 2025 i think), so keep at it and it’ll prob show up when they roll it out!
- Where i got this? Folks talking here + some github support hints (like needing PRs in multiple repos).

Heart on Your Sleeve
- What it is? Shows you’re all heart with the <3 reaction.
- How to get? 
  + Just slap a <3 on a comment, issue, pr, or discussion in a public repo.
  + Seems super easy, maybe one is enough? But its also in testing, so might not pop yet.
- Where i got this? Saw it in this repo’s chats and other threads.

Both are kinda unreleased still, per this repo’s notes, so don’t freak if they don’t show up right away. Github blog might say more later! Hope this helps—lemme know if u got more deets to share!

**Comment by castilla204** [comment]
cool
